src/test/java/TestMain.java

```diff
@@ -0,0 +1,62 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestMain {
+
+    @Test
+    public void shouldReturnTwentyThree() {
+        //given
+        final int givenForTestPurpose = 10;
+        final int expectedValue = 23;
+        //then
+        SumOfMultipleThreeAndFiveTogether.generateSumOfMultipleThreeAndFive(givenForTestPurpose);
```

Comment:
Ta metoda powinna zwracać jakiś rezultat który potem podasz do asercji w linii 15.

Reply:
Taki rezultat przypisz sobie do zmiennej i wykorzystaj w metodzie `assertThat`

Reply:
To samo tyczy się wszystkich metod niżej.